pc2cps

```diff
@@ -132,10 +188,11 @@ def main(args):
     parser.add_argument(
         'input', metavar='PACKAGE', type=str,
         help='Input pkg-config (.pc) file name or package name')
+    parser.add_argument("-n", "--library-name", required=False,
```

Comment:
The change in line wrapping style here vs. the previous line is a little jarring, as is the use of `"` for strings, which I don't see happening elsewhere. Also, the `.` at the end of the `help` is inconsistent.